build/template-OneBranch-libsandsamples.yaml

```diff
@@ -33,9 +33,5 @@ steps:
     configuration: ${{ parameters.BuildConfiguration }}
 
 # Run Post-build code analysis (e.g. Roslyn)
-- template: template-postbuild-code-analysis.yaml
```

Comment:
The analysis tools will be auto-injected in OneBranch then?

Reply:
yes